client/grpc-web-fake-transport/src/index.spec.ts

```diff
@@ -20,6 +21,7 @@ describe("FakeTransportBuilder", () => {
     it("should allow the response trailers to be stubbed", done => {
       const expectedTrailers = new grpc.Metadata({ foo: "bar" });
       const transport = new FakeTransportBuilder()
+        .withManualTrigger()
```

Comment:
Hmm, I don't understand what's going on here - could you please explain this change? I see you've made it be a manual trigger, but I don't see you triggering it anywhere in the test?